Missing hints in config commands for jackson

When executing config commands there is an error with native image.

```
$ spring-cli config list
Unable to read from path /home/jvalkealahti/.config/springcli/command-defaults.yml
java.lang.RuntimeException: Unable to read from path /home/jvalkealahti/.config/springcli/command-defaults.yml
	at org.springframework.cli.support.configfile.YamlConfigFile.read(YamlConfigFile.java:46)
	at org.springframework.cli.support.configfile.UserConfig.getConfig(UserConfig.java:60)
	at org.springframework.cli.support.SpringCliUserConfig.getCommandDefaults(SpringCliUserConfig.java:188)
	at org.springframework.cli.command.ConfigCommands.configList(ConfigCommands.java:117)
	at java.lang.reflect.Method.invoke(Method.java:566)
	at org.springframework.shell.command.invocation.InvocableShellMethod.doInvoke(InvocableShellMethod.java:308)
	at org.springframework.shell.command.invocation.InvocableShellMethod.invoke(InvocableShellMethod.java:233)
	at org.springframework.shell.command.CommandExecution$DefaultCommandExecution.evaluate(CommandExecution.java:151)
	at org.springframework.shell.Shell.evaluate(Shell.java:205)
	at org.springframework.shell.Shell.run(Shell.java:137)
	at org.springframework.shell.jline.NonInteractiveShellRunner.run(NonInteractiveShellRunner.java:104)
	at org.springframework.shell.DefaultShellApplicationRunner.run(DefaultShellApplicationRunner.java:65)
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:762)
	at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:752)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1306)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1295)
	at org.springframework.cli.SpringCliApplication.main(SpringCliApplication.java:30)
Caused by: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Cannot construct instance of `org.springframework.cli.support.SpringCliUserConfig$CommandDefaults` (no Creators, like default constructor, exist): cannot deserialize from Object value (no delegate- or property-based Creator)
 at [Source: (DataInputStream); line: 2, column: 1]
	at com.fasterxml.jackson.databind.DeserializationContext.reportBadDefinition(DeserializationContext.java:1904)
	at com.fasterxml.jackson.databind.DatabindContext.reportBadDefinition(DatabindContext.java:400)
	at com.fasterxml.jackson.databind.DeserializationContext.handleMissingInstantiator(DeserializationContext.java:1349)
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1415)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:351)
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:184)
	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:323)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4674)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3666)
	at org.springframework.cli.support.configfile.YamlConfigFile.read(YamlConfigFile.java:44)
	... 17 more

```

--- a/src/main/java/org/springframework/cli/config/SpringCliNativeConfiguration.java
+++ b/src/main/java/org/springframework/cli/config/SpringCliNativeConfiguration.java
@@ -54,6 +54,9 @@ import org.springframework.cli.initializr.model.JavaVersion.JavaVersionValues;
 import org.springframework.cli.initializr.model.Language.LanguageValues;
 import org.springframework.cli.initializr.model.Packaging.PackagingValues;
 import org.springframework.cli.initializr.model.ProjectType.ProjectTypeValue;
+import org.springframework.cli.support.SpringCliUserConfig.CommandDefault;
+import org.springframework.cli.support.SpringCliUserConfig.CommandDefaults;
+import org.springframework.cli.support.SpringCliUserConfig.Option;
 import org.springframework.nativex.hint.FieldHint;
 import org.springframework.nativex.hint.JdkProxyHint;
 import org.springframework.nativex.hint.MethodHint;
@@ -254,6 +257,15 @@ import org.springframework.nativex.type.NativeConfiguration;
 			types = {
 				GHMyself.class, GHObject.class, GHPerson.class, GHUser.class, GHCommit.class, GHLicense.class,
 				GHRepository.class, GHVerification.class, GitUser.class
+			},
+			access = {
+				TypeAccess.PUBLIC_CONSTRUCTORS, TypeAccess.DECLARED_CLASSES, TypeAccess.DECLARED_CONSTRUCTORS,
+				TypeAccess.DECLARED_FIELDS, TypeAccess.DECLARED_METHODS
+			}
+		),
+		@TypeHint(
+			types = {
+				CommandDefaults.class, CommandDefault.class, Option.class
 			},
 			access = {
 				TypeAccess.PUBLIC_CONSTRUCTORS, TypeAccess.DECLARED_CLASSES, TypeAccess.DECLARED_CONSTRUCTORS,

